docs/legacy-protocols/how-to-authenticate-an-imap-pop-smtp-application-by-using-oauth.md

```diff
@@ -87,6 +87,10 @@ dXNlcj10ZXN0QGNvbnRvc28ub25taWNyb3NvZnQuY29tAWF1dGg9QmVhcmVy
 IEV3QkFBbDNCQUFVRkZwVUFvN0ozVmUwYmpMQldaV0NjbFJDM0VvQUEBAQ==
 ```
 
+### SASL XOAUTH2 authentication for shared mailbox in Office 365
+
+In case of shared mailbox access using OAuth, user needs to obtail it´s access token as for users mailbox access but replace the userName in the SASL XOAUTH2 encoded string with the email address of the shared mailbox. This will allow user to successfully authenticate and access the shared mailbox using it own access token. 
```

Comment:
Please replace this with "In case of shared mailbox access using OAuth, application needs to obtain the access token on behalf of a user but replace the userName field in the SASL XOAUTH2 encoded string with the email address of the shared mailbox. "